metadata/Dev-Corner.yml

```diff
@@ -152,6 +152,24 @@ material:
     - "Requires understanding of both the Galaxy ecosystem as well as HTML5/JS"
     - "Performance is more important than for static Galaxy tools"
 
+-
+    title: "BioBlend module, a python library to use Galaxy API"
+    type: "tutorial"
+    name: "bioblend_api"
+    zenodo_link: ""
+    galaxy_tour: ""
+    hands_on: "no"
+    slides: "yes"
+
+    objectives:
+    - "Get familiar with BioBlend with ipython"
+    - "Launch a Galaxy job / Visualize your actions with Galaxy"
+    - "Launch a Galaxy workflow / Visualize your actions with Galaxy"
+
```

Comment:
Can you add the questions and the key_points, please? They are part of the templating and will be empty otherwise. Thanks